### PR TITLE
Ultrabark

### PR DESCRIPTION
**Pull request by jeznag**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -16,12 +16,8 @@ const DIALOG_STAGES = [
         '😱 you are late.',
     ],
     VERBAL_ALERTS = [
-        'yellow alert, meeting imminent',
-        'red alert',
-        "heyyoo... it's time",
-        'beep beep!',
-        'meeting time!',
-        'hey!! check your calendar!',
+        'spotify:track:7jL5aGalMaZRfnJ9jasABT',
```
Comment (jeznag) [inline config.js:19]: probably should revert

File: events.json.sample [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    }
+  ],
+  "eventKeywordsToIgnore": ["no-notify", "focus", "blocked"],
+  "voiceOptions": {
+    "using": "Karen",
+    "speakingRate": 150,
+    "pitch": 50,
+    "volume": 0.5,
+    "modulation": 2,
+    "waitForCompletion": true
```
Comment (jeznag) [inline events.json.sample:17]: Can you provide constraints for these properties so we know what fields to put in the settings UI?

File: config.js (hunk 1 above)
Comment (jeffdechambeau) [reply config.js:19]: Sure. What about migrating this into the events.json file? That would similarly expose it to the UI as per your other comment.

File: events.json.sample (hunk 2 above)
Comment (jeffdechambeau) [reply events.json.sample:17]: Done

File: config.js (hunk 1 above)
Comment (jeznag) [reply config.js:19]: good idea.

**Pull request merged**